x-pack/plugins/uptime/common/runtime_types/monitor_management/config_key.ts

```diff
@@ -19,7 +19,9 @@ export enum ConfigKey {
   MAX_REDIRECTS = 'max_redirects',
   METADATA = '__ui',
   MONITOR_TYPE = 'type',
+  ID = 'id',
```

Comment:
As far as I'm aware, id is not being saved directly on the monitor since it comes from the saved object.